linker-interface/shared/src/main/scala/org/scalajs/linker/interface/StandardConfig.scala

```diff
@@ -110,6 +110,22 @@ final class StandardConfig private (
   def withBatchMode(batchMode: Boolean): StandardConfig =
     copy(batchMode = batchMode)
 
+  def serialized: String =
```

Comment:
Consider:

- Calling this `fingerprint`? (`serialized` kind of does imply that we can deserialize it...).
- Return an `Array[Byte]` (not 100% sure about this, but something to think about).
- Putting this on the companion (feels cleaner to me, probably because it separates the namespace of config values and operations).

Reply:
- I like `fingerprint` a lot. I've renamed all instances to that.
- I'm not convinced by `Array[Byte]` just yet. I have a few concerns that I'll address in the top-level conversation below.
- I agree. Done :)